CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 # spacebench (developing)
 
+## Next release
```

Comment:
Using the term `developing` implies that the code will be part of the upcoming release. Please categorize your new contributions as `changed`, `added`, or `removed`.

Reply:
Just pushed a new version.